src/test/java/com/fasterxml/jackson/integtest/messgepack/MessagePackExtensionTest.java

```diff
@@ -0,0 +1,20 @@
+package com.fasterxml.jackson.integtest.messgepack;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.POJONode;
+import com.fasterxml.jackson.integtest.BaseTest;
+import org.msgpack.jackson.dataformat.MessagePackExtensionType;
+
+import java.util.List;
+
+public class MessagePackExtensionTest extends BaseTest {
+    public void testPOJONodeWithMessagePackExtensionShouldNotThrowExceptionWhenEvaluate() {
```

Comment:
One request: could you add a comment before this line, with URL to issue that triggered addition (in case someone wants to have a look)?

Reply:
of course :)